docs/beamrunner/beamrunner-5-ref.md

```diff
@@ -98,3 +98,6 @@ The Streams Runner package contains the following folders:
 
 <br>
 For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on the Beam application command line.
+
+## SDK API Reference
+See [javadoc](../beamrunner/release/1.0/javadoc/index.html)

```

Comment:
There needs to be a period at the end of this sentence. Also, we're typically encouraged to include more context for the link. For example:
"For more information about the `com.ibm.streams.beam.sdk.io` package, see [javadoc](../beamrunner/release/1.0/javadoc/index.html)."